src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -6966,6 +6966,52 @@ address generate_avx_ghash_processBlocks() {
       StubRoutines::_montgomerySquare
         = CAST_FROM_FN_PTR(address, SharedRuntime::montgomery_square);
     }
+#ifdef __VECTOR_API_MATH_INTRINSICS_COMMON
+    void *libsvml = NULL;
+    char ebuf[1024];
+    libsvml = os::dll_load(JNI_LIB_PREFIX "svml" JNI_LIB_SUFFIX, ebuf, sizeof ebuf);
+    if (libsvml != NULL) {
+      log_info(library)("Loaded library %s, handle " INTPTR_FORMAT, JNI_LIB_PREFIX "svml" JNI_LIB_SUFFIX, p2i(libsvml));
+      if (UseAVX > 2) {
```

Comment:
Please add comment here explaining naming convention you are using here. 
What `f16_ha_z0' mean? Why `8_ha_z0` and not `d8_ha_z0`?
What is `l9`, `e9`, `ex`?